comdb2/cdb2.py

```diff
@@ -128,48 +128,79 @@
     Make sure to carefully read :ref:`String and Blob Types` on that page.
 """
 
+from __future__ import annotations
+
+import datetime
+from typing import Any, Callable, Iterator, List, Mapping, Tuple, Union
 from ._cdb2_types import Error, Effects, DatetimeUs
 from ._ccdb2 import Handle as CHandle
 
-__all__ = ['Error', 'Handle', 'Effects', 'DatetimeUs',
-           'ERROR_CODE', 'TYPE', 'HANDLE_FLAGS']
+__all__ = [
+    "Error",
+    "Handle",
+    "Effects",
+    "DatetimeUs",
+    "ERROR_CODE",
+    "TYPE",
+    "HANDLE_FLAGS",
+]
+
+Value = Union[
```

Comment:
I would prefer a more explicit name here, but I guess this was already called this in the `.pyi`, and part of our public interface… perhaps there is nothing we can do here.

Reply:
Well - I guess we could pick a different name... up until now it hasn't been part of our _real_ public interface, just our types. We could conceivably get away with a new name plus a type alias. What name would you prefer, if we were to change it?

Reply:
I think I'd call this a `RowValue` instead